projects/pgai/README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+    <img height="200" src="https://github.com/timescale/pgai/blob/main/docs/images/pgai_logo.png" alt="pgai"/>
```

Comment:
Why fixing a heigh here?
```suggestion
    <img src="https://github.com/timescale/pgai/blob/main/docs/images/pgai_logo.png" alt="pgai"/>
```

Reply:
I copy pasted from the original README.

Reply:
Looks way better without that restriction in all resolutions